internal/handlers/sqlite/msg_create.go

```diff
@@ -79,6 +70,44 @@ func (h *Handler) MsgCreate(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 		return nil, err
 	}
 
+	params := backends.CreateCollectionParams{
+		Name: collectionName,
+	}
+
+	capped, err := common.GetOptionalParam[bool](document, "capped", false)
+	if err != nil {
+		return nil, err
+	}
+
+	if capped {
+		// size param is required
+		var size any
+
+		size, err = document.Get("size")
+		if err != nil {
+			return nil, err
+		}
+
+		params.CappedSize, err = commonparams.GetWholeNumberParam(size)
+		if err != nil {
+			return nil, err
+		}
+
+		if params.CappedSize%256 != 0 {
+			params.CappedSize = (params.CappedSize/256 + 1) * 256 // there is a requirement to round up to 256 bytes' multiplier
```

Comment:
```suggestion
			params.CappedSize = (params.CappedSize/256 + 1) * 256
```

Comments don't describe _what_ we do – the fact that we round a number is clear from the code. Comment should describe _why_. The best that comment could say is "we do this because MongoDB does the same", but that comment could be added to every FerretDB code line. So let's just remove it.

That is described in CONTRIBUTING.md for quite some time now